packages/core/src/App/Containers/Layout/header/default-header.jsx

```diff
@@ -91,6 +93,32 @@ const DefaultHeader = ({
             }
             return true;
         });
+
+    const Divider = () => {
+        return <div className='header__menu--separator' />;
+    };
+
+    const ExploreTradingHub = () => {
+        const platform_store = React.useContext(PlatformContext);
+        const EnablePreAppstore = () => {
+            platform_store.setIsPreAppStore(true);
+        };
+        return (
+            <div className='header__menu__redirect'>
+                <BinaryLink
+                    to={routes.trading_hub}
+                    className='header__menu__redirect--link'
+                    onClick={EnablePreAppstore}
+                >
+                    <Text as='p' size='xs' color='general'>
```

Comment:
color is general by default
```suggestion
                    <Text as='p' size='xs'>
```